src/emulator/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -77,53 +90,43 @@ EXPORT(int, sceDisplayUnregisterVblankStartCallback) {
 EXPORT(int, sceDisplayWaitSetFrameBuf) {
     STUBBED("move after setframebuf");
 
-    if (!host.display.sync_rendering) {
-        std::unique_lock<std::mutex> lock(host.display.mutex);
-        host.display.condvar.wait(lock);
-    } else
-        host.display.condvar.notify_all();
-
-    if (host.display.abort.load())
-        return SCE_DISPLAY_ERROR_NO_PIXEL_DATA;
-
-    return SCE_DISPLAY_ERROR_OK;
+    return display_wait(host);
```

Comment:
same

Reply:
Is already here before that, i prefere just separe message stub and return function for more, clear

Reply:
oh i mean line 79-92

Reply:
yes, and like i have said, i added nothing, i have just remplaced return by news funcction, so here is same master.
Stub info
Spacing
return